src/FieldWarning/Assets/Units/Component/Movement/Pathfinder.cs

```diff
@@ -230,13 +287,13 @@ private void UpdateWaypoint()
             }
         }
 
-        private void UpdateWaypointAngle()
+        private void UpdateWaypointAngle(Vector3 currentPosition)
```

Comment:
If you happen to know what this method does, a comment about its purpose would be great :D